execution_engine/src/core/runtime/mod.rs

```diff
@@ -1787,6 +1787,33 @@ where
         Ok(Ok(()))
     }
 
+    fn enable_contract_version(
+        &mut self,
+        contract_package_hash: ContractPackageHash,
+        contract_hash: ContractHash,
+    ) -> Result<Result<(), ApiError>, Error> {
+        let contract_package_key = contract_package_hash.into();
+        self.context.validate_key(&contract_package_key)?;
+
+        let mut contract_package: ContractPackage = self
+            .context
+            .get_validated_contract_package(contract_package_hash)?;
+
+        // Return an error in trying to disable the (singular) version of a locked contract.
```

Comment:
```suggestion
        // Return an error if trying to enable the (singular) version of a locked contract.
```
Although I'd be inclined to remove the comment altogether. I'm not sure what point it's trying to make - the "singular" seems important, but isn't really I think?